Makefile

```
@@ -681,7 +691,7 @@ $(TESSTRAIN_EXECUTABLES:$(BIN)/%=%): %: $(BIN)/%
 clean: clean-tesseract
 .PHONY: clean-tesseract
 clean-tesseract:
-	$(RM) -r $(VIRTUAL_ENV)/build/tesseract
+	-$(RM) -r $(VIRTUAL_ENV)/build/tesseract
```

This change is not needed.

---

It is: `make clean` (which depends on `clean-tesseract`) should work, regardless of whether Tesseract has already been built. This is common practice. (No-one expects `clean` to fail if there was nothing to clean.)

---

The old code does not fail for me. As long as `RM` is defined to be `rm -f` I would be surprised if it failed for you.

---

It's not necessary to ignore errors from `rm -rf` because it will virtually never throw one. But it doesn't hurt either, making clear the intent to "remove no matter what" which fits the target.